src/components/sequencing/SequenceEditor.svelte

```diff
@@ -407,115 +415,139 @@
   function toggleSeqJsonEditor(): void {
     toggleSeqJsonPreview = !toggleSeqJsonPreview;
   }
+
+  function formatDocument() {
+    if (isInVmlMode) {
+      vmlFormat(editorSequenceView);
+    } else {
+      seqNFormat(editorSequenceView);
+    }
+  }
+
+  function inVmlMode(sequenceName: string | undefined): boolean {
+    return !!sequenceName?.endsWith('.vml');
```

Comment:
Do you mind handling the `undefined` case explicitly? I know it takes 2 seconds to understand this, but seeing `undefined` explicitly mapping to `false` would be a little clearer rather than the double negation just for the `undefined` case.

Reply:
addressed in https://github.com/NASA-AMMOS/aerie-ui/pull/1472/commits/1e966d2de7457672427eb7b549b54fc20dc85284